src/routes/dashboard/+page.svelte

```diff
@@ -20,9 +33,13 @@
 
 <div class="dashboard">
   <h1>Dashboard</h1>
-  <Balances />
-  <Streams />
-  <Splits />
+  {#await loading}
+    loading...
+  {:then}
+    <Balances />
+    <Streams />
+    <Splits {userId} />
+  {/await}
```

Comment:
Could the splits section accept an `undefined` `userId` and internally wait for it to be populated? You could pass `loaded = false` to `SectionSkeleton` in order to display a loading state only for the Splits section. That would avoid having another loading state above the sections, which already each have their own section-level loading state.

Reply:
Ok yep, [removed](https://github.com/radicle-dev/drips-app-v2/pull/36/commits/396bc452780a1c989116d712ceb3e994c070d3b9)